docs/getting-started/integrate-auth/25_react.mdx

```diff
@@ -41,74 +50,84 @@ import OryCLI from '../_common/ory-cli.mdx'
-they're different hostnames.
+Go to [localhost:5173](http://localhost:5173/) to access your application.
 
-Using both interchangeably in your code can cause problems with the origin of HTTP cookies sent by the application. When that
-happens Ory APIs might not be able to read the cookies sent by your app.
+## Make authenticated calls to your API
 
-:::
+To make authenticated requests to your API there are two main components:
 
-To get your application that runs locally and Ory APIs on the same domain, use Ory Tunnel - a development tool bundled with Ory
-CLI. It's like a microservice - an authentication API server on your domain!
+- When making AJAX requests you must set `{"credentials": "include"}` in the options. For the `fetch` method it looks like this:
 
-To get your project's SDK URL, go to <ConsoleLink route="project.settings" />, and copy the URL from the **API Endpoints** field.
+```mdx-code-block
+import AppWithAPI from '!!raw-loader!../../../code-examples/protect-page-login/react/src/AppWithApi.tsx'
 
-```shell
-export ORY_SDK_URL=https://$PROJECT_SLUG.projects.oryapis.com
+<CodeBlock language="tsx" title="./src/AppWithAPI.tsx">{AppWithAPI}</CodeBlock>
 ```
 
-In a new terminal window, start the Ory Tunnel. Upon first start, the Ory Tunnel will ask you to log into your Ory Console
-account.
+- Your API must have a CORS middleware with `credentials: true` and `Access-Control-Allow-Origin` of your frontend app (here
+  `http://localhost:5173`).
 
-```shell
-npx @ory/cli tunnel --dev http://localhost:3000
-```
-
-Ory APIs are now mirrored on `http://localhost:4000`. Use that URL as the `baseUrl` for the `@ory/client` SDK (see code above).
-The `--dev` flag disables security checks for easier integration and should not be used when deploying the Tunnel to a staging
-environment.
+Let's put this into action. Create a simple HTTP API with express. Run:
 
-## Run your React app
+```shell-script
+mkdir api
+cd api
+npm init
+npm i --save @ory/client-fetch express cors
+```
 
-Great, that's it. Let's run your application. Set up your environment variables to connect with Ory's APIs
+Next, create a simple API in `index.js`:
 
 ```mdx-code-block
-import SdkEnvVar from '@site/src/components/SdkEnvVar'
+import handlerApi from '!!raw-loader!../../../code-examples/auth-api/expressjs/index.js'
 
-<SdkEnvVar />
+<CodeBlock language="js" title="./server/index.js">{handlerApi}</CodeBlock>
 ```
 
-and start the React development server:
+Finally, start the server:
 
 ```shell
-npm start
+node index.js
 ```
 
-Then open [http://localhost:3000](http://localhost:3000) in your browser. After loading, you now are presented with Ory's Sign In
-page! Let's click on sign up and create your first user.
+### See it in action
+
+Access your react app at [localhost:5173](http://localhost:8081/), sign in, and see if you can make authenticated requests to your
```

Comment:
```suggestion
Access your react app at [localhost:5173](http://localhost:5173/), sign in, and see if you can make authenticated requests to your
```

I think there is a typo here? 
What exactly should I do here as a user - create an account? / how do I make an authenticated request to the API? its not really obvious to me from this part of the guide.

For example when I sign-up I get redirect to localhost:4000 with an error 
```
{"error":{"code":"404","details":{"ruleId":"6176b3cb"},"message":"Invalid path","reason":"No resource found using the specified path.","status":"Not Found"}}
```

I think it would be good to include very specific instructions here, that I can follow "braindead" and also a curl command to test the API auth part.